packages/tasks/src/model-libraries.ts

```diff
@@ -102,6 +102,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.bertopic,
 		filter: true,
 	},
+	big_vision: {
```

Comment:
dash or underscore?

Reply:
yep sounds good, but note that if they ever publish to PyPI, it will become `big-vision` :)

Reply:
Yes, although as a research repo I think they might never publish there hehe